### PR TITLE
Img and video improvements

### PR DESCRIPTION
**Pull request by supdood**
Made improvements to both video and image elements in the battle index view and individual battle views

### CODE REVIEW

File: app/views/battles/show.html.haml [hunk 1]
```diff
@@ -13,7 +13,7 @@
       - if @battle.won?
         .columns.small-2
           %strong WINNER:
-          = image_tag @battle.winning_comment.image_url
+          = text_with_images @battle.winning_comment.image, true
```
Comment (BFGeorge9000) [inline app/views/battles/show.html.haml:16]: Do we need to pass `true` as a second argument here, given that it is the default value for the `flex` parameter in `text_with_images`

File: app/helpers/battles_helper.rb [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
     video_regex = /https?:\/\/\S+\.(?:gifv|webm|mp4)/i
     image_regex = /https?:\/\/\S+\.(?:gif|jpg|jpeg|png)/i
     text.split(/(?<=\s)/).map do |match|
       if match.match(video_regex)
-        substitution = match.sub(/gifv$/, "webm")
-        "<br>#{video_tag(substitution, autoplay:"autoplay", loop:"loop")}<br>"
+        if flex
+          substitution = match.sub(/gifv$/, "webm")
+          "<div class='flex-video'>#{video_tag(substitution, autoplay:"autoplay", loop:"loop")}</div>"
+        else
```
Comment (BFGeorge9000) [inline app/helpers/battles_helper.rb:10]: Do we ever call `text_with_images([some image], false)`?

Comment (supdood) [inline app/helpers/battles_helper.rb:10]: @BFGeorge9000 Oops, the default is supposed to be false.  Which explains the reason why I called it specifying "true" in the show view.

**Comment by supdood** [comment]
Ok, I think it is good to go now.

**Comment by BFGeorge9000** [comment]
:+1:

**Pull request merged**